packages/cli/test/fixtures/copyright/single-package/src/application.ts

```diff
@@ -1,6 +1,6 @@
-// Copyright IBM Corp. 2020. All Rights Reserved.
+// Copyright ACME Inc. 2020. All Rights Reserved.
```

Comment:
Is this change intentional?

Reply:
@mayank-SFIN571 I am not sure why I pushed this, but I already fixed it.